ISS/user_reg.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render
+from django.http import HttpResponseRedirect    
+from django.contrib import auth                 
+from django.core.context_processors import csrf 
+from rforms import MyRegistrationForm
+
+def register_user(request):
+    if request.method == 'POST':
+        form = MyRegistrationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            return HttpResponseRedirect('/accounts/register_success')
+    args = {}
+    args.update(csrf(request))
+    args['form'] = MyRegistrationForm()
+    print args
+    return render(request, 'register.html', args)
```

Comment:
There doesn't seem to be a register.html template in this PR, is that part of the auth app?